appcelerator/titanium_mobile
# [TIMOB-26329] Android: Modified build to generate missing "R" classes for libraries added in 7.3.0

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-26329

**Test:**
1. Download "hyperloop-examples" project from [here](https://github.com/appcelerator/hyperloop-examples).
2. Build and run on Android.
3. Tap on the "Native Layout (XML)" row in the app.
4. Verify that the app does not crash.
